exporter_test.go

```diff
@@ -811,6 +812,7 @@ SAVE MYSQL USERS TO DISK;
 			for k := range m.labels {
 				cv.So(k, convey.ShouldEqual, strings.ToLower(k))
 			}
+			fmt.Println(m)
```

Comment:
Was this for debugging though? Or something to keep? (Asking since it doesn't seem to use a regular logger either)

Reply:
Fixed